### PR TITLE
Support currency and from/to dates in report

### PR DESCRIPTION
**Pull request by bkutil**
Adds support for the following camt.052 Report elements:

- Report account currency (`Rpt/Acct/Ccy`)
- Report from/to date times (`Rpt/FrToDt/FrDtTm` and `Rpt/FrToDt/ToDtTm`)

### CODE REVIEW

File: spec/fixtures/052/valid_example.xml [hunk 1]
```diff
@@ -14,6 +14,10 @@
     <Id>0352C5220131227110203</Id>
     <ElctrncSeqNb>130000005</ElctrncSeqNb>
     <CreDtTm>2013-12-27T11:01:46.0+01:00</CreDtTm>
+    <FrToDt>
+      <FrDtTm>2009-01-01T00:00:00:0+01:00</FrDtTm>
```
Comment (tobischo) [inline spec/fixtures/052/valid_example.xml]: This does not look like a valid value
Shouldn't this be `2009-01-01T00:00:00.0+01:00`

File: spec/fixtures/052/valid_example.xml [hunk 2]
```diff
@@ -14,6 +14,10 @@
...
+      <ToDtTm>2013-12-01T00:00:00:0+01:00</ToDtTm>
```
Comment (tobischo) [inline spec/fixtures/052/valid_example.xml]: Same here

File: spec/lib/camt_parser/052/report_spec.rb [hunk 3]
```diff
@@ -15,4 +15,7 @@
   specify { expect(ex_rpt.closing_balance).to be_kind_of(CamtParser::AccountBalance) }
 
   specify { expect(ex_rpt.identification).to eq("0352C5220131227110203") }
+
+  specify { expect(ex_rpt.from_date_time).to be_kind_of(Time) }
```
Comment (tobischo) [inline spec/lib/camt_parser/052/report_spec.rb]: Would you mind also testing that the value was parsed correctly?
I would also like to see the `nil` case tested, which is handled in the method.

Comment (bkutil) [reply spec/lib/camt_parser/052/report_spec.rb]: Added a check for the `Time` value.

As for the `nil` case, what would be the best way to implement it?

Comment (bkutil) [reply spec/lib/camt_parser/052/report_spec.rb]: OK, I kept the original fixture intact - the `nil` checks run against that, and added a new fixture for parsing checks. I used my (anonymized/changed) bank export...

**Comment by tobischo** [comment]
Released as v2.8.0

**Pull request merged**